packages/message-relayer/src/service.ts

```diff
@@ -166,17 +211,25 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
         const relayer = await this.state.Lib_AddressManager.getAddress(
           'OVM_L2MessageRelayer'
         )
+        //that's set in the MMR deploy
+
+        //console.log('\nRelayer:\n', relayer)
         // If it is address(0), then message relaying is not authenticated
         if (relayer !== ethers.constants.AddressZero) {
+          // relaying is authenticated
+          // so the wallet address has to be the same as whatever is set in
+          // OVM_L2MessageRelayer?
           const address = await this.options.l1Wallet.getAddress()
-          if (relayer !== address) {
-            throw new Error(
-              `OVM_L2MessageRelayer (${relayer}) is not set to message-passer EOA ${address}`
-            )
-          }
+          //console.log('\nAddress:\n', address)
+          // if (relayer !== address) {
```

Comment:
should we keep this check? the wallet address is from env?

Reply:
Need to figure out - this check breaks as soon as we do this

```  
//OVM_L2MessageRelayer *must* be set to multi message relayer address on mainnet.
  //if (hre.network.name.includes('mainnet')) {
  const OVM_L1MultiMessageRelayer = await getDeployedContract(
    hre,
    'OVM_L1MultiMessageRelayer'
  )

  await registerAddress({
    hre,
    name: 'OVM_L2MessageRelayer',
    address: OVM_L1MultiMessageRelayer.address,
  })
  //}```

The issues is that for everything to work, contract wise, `OVM_L2MessageRelayer` needs to be set to `OVM_L1MultiMessageRelayer.address`. But the message-relayer check is checking for 
``OVM_L2MessageRelayer (${relayer}) is not set to message-passer EOA ${address}`. That works fine in normal non-btach mode, but then breaks when `OVM_L2MessageRelayer` is set to `OVM_L1MultiMessageRelayer.address`